code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -215,6 +215,12 @@
 				else if(E.status & ORGAN_SPLINTED)
 					wound_flavor_text["[E.limb_name]"] = "[p_they(TRUE)] [p_have()] a splint on [p_their()] [E.name]!\n"
 
+			if(E.open)
+				if(E.is_robotic())
+					msg += "<b>The maintenance hatch on [p_their()] [ignore_limb_branding(E.limb_name)] is open!</b>\n"

```

Comment:
Wouldn't it be better to use span classes for these?

Reply:
I'm not sure if it it's better to do `<span class='bold'>` or just use `<b>` to be honest. I see both used in this file.

Reply:
I thought generally we prefer using span classes, but would have to ask a maint what the actual policy is.